File: C#/Task240.cs
using System;

public class Task240
{
    public static void Main()
    {
        double x, u, s = 1, eps = 0.00001;

        Console.WriteLine("Введіть значення x");
        x = double.Parse(Console.ReadLine());

        u = x / 4;
        int n = 2;

        while (Math.Abs(u) > eps)
        {
            s += u;
            u *= -x * (4 * n - 5) / (4 * n);
            n++;
        }

        Console.WriteLine($"Значення суми s = {s:F6}");
        Console.WriteLine($"Значення функції: {Math.Pow(1 + x, 0.25):F6}");
        Console.WriteLine("Кількість доданків " + (n - 1));
    }
}

File: Python/Task240.py
eps = 0.00001
print('Веедіть значення змінної x')
x = float(input())
s = 1; n = 2; u = x / 4
while abs(u) > eps: 
    s += u
    u *= -x * (4 * n - 5) / (4 * n)
    n += 1
f = (1 + x)**0.25
print('Сума ряду s = %12.5f' % s)
print('Значення функції f = %12.5f' % f)
print('Кількість доданків : %5d'% (n - 1))
